gremlin-javascript/src/main/javascript/gremlin-javascript/lib/structure/graph.js

```diff
@@ -158,11 +159,20 @@ function areEqual(obj1, obj2) {
   return false;
 }
 
+/**
+* Returns an anonymous traversal source.
+* @returns {GraphTraversalSource}
+*/
+function traversal() {
```

Comment:
This method is being implemented on `./lib/structure/graph.js` when it should be under `lib/process` directory.